draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -243,27 +266,56 @@ steps, taking the `stream` as input.
 1. If reading 10 bytes from `stream` returns an error or doesn't return the
    bytes with hex encoding "84 48 F0 9F 8C 90 F0 9F 93 A6" (the CBOR encoding of
```

Comment:
Oh, I had overlooked this... Now the top-level array has 6 elements, so this should be "86 48 ...".

Reply:
Thanks! https://github.com/WICG/webpackage/pull/454